nodebuilder/header/flags.go

```diff
@@ -27,16 +24,16 @@ func HeadersFlags() *flag.FlagSet {
 	return flags
 }
 
-// ParseHeadersFlags parses Header package flags from the given cmd and applies values to Env.
-func ParseHeadersFlags(ctx context.Context, cmd *cobra.Command, cfg *nodebuilder.Config) (context.Context, error) {
-	if ctx, err := ParseTrustedHashFlags(ctx, cmd, cfg); err != nil {
-		return ctx, err
+// ParseFlags parses Header package flags from the given cmd and saves them to the passed config.
```

Comment:
```suggestion
// ParseFlags parses Header package flags from the given cmd and applies them to the passed config.
```

we don't necessarily *save* the modified conf, only on Init